### PR TITLE
Create new project page

### PR DESCRIPTION
**Pull request by AbubakarImam**
created a new page to create a new project

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -6,12 +6,14 @@ import { BrowserModule } from '@angular/platform-browser';
...
 import { AppComponent } from './app.component';
 import { HomeComponent } from './pages/home/home.component';
+import { NewListComponent } from './pages/new-list/new-list.component';
 // Import other modules and components
 
 @NgModule({
   declarations: [
     AppComponent,
     HomeComponent,
+    NewListComponent,
```
Comment (ErikNaljota) [inline src/app/app.module.ts]: Unnecessary empty line

File: src/app/app.module.ts [hunk 2]
```diff
@@ -6,12 +6,14 @@ import { BrowserModule } from '@angular/platform-browser';
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
 import { HomeComponent } from './pages/home/home.component';
+import { NewListComponent } from './pages/new-list/new-list.component';
 // Import other modules and components
```
Comment (ErikNaljota) [inline src/app/app.module.ts:9]: Unnecessary comment

File: src/app/pages/home/home.component.html [hunk 3]
```diff
@@ -1,11 +1,18 @@
...
+<div class="centered-content">
     <div class="flex w-full h-full max-h-96 justify-center">
         <div class="bg-white w-64 flex flex-col py-2 rounded-l-md relative">
             <h1 class="text-black p-2 font-extrabold">Projects</h1>
             <div class="list-menu container flex items-center flex-col flex-grow-1">
-                <a class="item flex items-center justify-center px-2 py-4 rounded-md w-full text-[#5f5f5f] mb-1
+                <!-- <a class="item flex items-center justify-center px-2 py-4 rounded-md w-full text-[#5f5f5f] mb-1
                      hover:bg-pink-400 active:bg-pink-600 active:text-red-200">
                     <p>Project 1</p>
+                </a> -->
```
Comment (ErikNaljota) [inline src/app/pages/home/home.component.html:9]: Should never leave commented code in pull requests if there is not a good reason for it

File: src/app/pages/home/home.component.ts [hunk 4]
```diff
@@ -1,10 +1,18 @@
-import { Component } from '@angular/core';
-
+import { Component, OnInit } from '@angular/core';
+import { Project } from 'src/models/project.model';
+import { ProjectService } from 'src/services/project.service';
 @Component({
```
Comment (ErikNaljota) [inline src/app/pages/home/home.component.ts]: Have one empty line between imports and `@Component`

File: src/app/pages/home/home.component.ts [hunk 5]
```diff
@@ -1,10 +1,18 @@
...
   selector: 'app-home',
   templateUrl: './home.component.html',
   styleUrls: ['./home.component.css']
 })
-export class HomeComponent {
+export class HomeComponent implements OnInit {
+  projects: Project[] = [];
```
Comment (ErikNaljota) [inline src/app/pages/home/home.component.ts:11]: Initialize variables in constructor

File: src/app/pages/new-list/new-list.component.ts [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+  createNewProject(title: string) {
+    this.projectService.addProject(title);
+    this.router.navigate(['']);
+
+    // const newProject = { name: title }
+    // const projects = [newProject];
+    // console.log(projects);
+
+    // this.router.navigate([''])
+    // clearInputField();
```
Comment (ErikNaljota) [inline src/app/pages/new-list/new-list.component.ts]: Don't send commented code to PR

File: src/services/project.service.ts [hunk 7]
```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+import { Project } from '../models/project.model';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class ProjectService {
+    private project: Project[] = [];
+
+    addProject(name: string) {
```
Comment (ErikNaljota) [inline src/services/project.service.ts]: Add access modifier and return type to all your functions
```suggestion
    public addProject(name: string): void {
```

**Review by ErikNaljota — CHANGES_REQUESTED** [review]

**Review by AbubakarImam — COMMENTED** [review]
resolved all the suggestions

**Review by Emperorchi — APPROVED** [review]

**Pull request merged**